pandas/core/reshape/pivot.py

```diff
@@ -422,13 +423,18 @@ def _convert_by(by):
 
 @Substitution("\ndata : DataFrame")
 @Appender(_shared_docs["pivot"], indents=1)
-def pivot(data: "DataFrame", index=None, columns=None, values=None) -> "DataFrame":
+def pivot(
+    data: "DataFrame",
+    index: Optional[Union[Axes, Sequence[Optional[Axes]]]] = None,
```

Comment:
I don’t think `Axes` is correct - were these not working with `Label`?

Reply:
emm, no, not working with `Label`, there is constructor inside `pivot`, and based on the signature of frame constructor, i think it is the reason why setting to `Axes` works @WillAyd 

```python
data=None,
index: Optional[Axes] = None,
columns: Optional[Axes] = None,
```

Reply:
Can you share the actual error this is causing? Although the keywords are the same the arguments should be different than the frame constructor I would think